monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+    thresholds=THRESHOLDS["queue_ops"],
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Operations queued due to a lock.",
+)
+
+# Additional Performance Panels
+op_queue_size = mongodb_timeseries(
+    "Operation queue size due to locks",
+    [
+        Target(
+            expr=f'sum by(job) (mongodb_ss_globalLock_currentQueue{{count_type=~"readers|writers", job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}})',
+            legendFormat="{{job}}",
+        )
+    ],
+    unit="none",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    valueMin=0,
+    description="Operations queued due to a lock.",
+)
+
+write_conflicts = mongodb_timeseries(
+    "Write Conflicts per Second",
+    [
+        Target(
+            expr=f'sum by(job) (\n  rate(mongodb_mongod_metrics_operation_total{{job=~"{Jobs.shard_pattern}",state="writeConflicts"}}[$__rate_interval])\n) / sum by(job) (\n  rate(mongodb_ss_opcounters{{job=~"{Jobs.shard_pattern}", legacy_op_type=~"insert|update|delete"}}[$__rate_interval])\n)',
+            legendFormat="Write conflict",
+        )
+    ],
+    # percent 0-100, NOT percent_unit
+    unit=UNITS.PERCENT_FORMAT,
+    thresholds=THRESHOLDS["write_conflicts"],
+    fillOpacity=27,
+    valueMax=1,
+    showPoints="auto",
+    description="The ratio of write conflicts to the total number of write operations. A value of 0% means no write conflicts occurred, while a value of 100% means all write operations were conflicts.",
+)
+
+reads_writes_locks = mongodb_timeseries(
+    "Reads & Writes clients locks",
+    [
+        Target(
+            expr=f'avg(mongodb_ss_globalLock_activeClients_{client_type}{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}})',
+            legendFormat=f"Active {client_type.title()}",
+        )
+        for client_type in ["readers", "writers"]
+    ]
+    + [
+        Target(
+            expr=f'avg(mongodb_ss_globalLock_currentQueue{{job=~"{Jobs.shard_pattern}",count_type="{client_type}", rs_nm=~"$rs_nm"}})',
+            legendFormat=f"Queued {client_type.title()}",
+        )
+        for client_type in ["readers", "writers"]
+    ],
+    unit="none",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    valueMin=0,
+    description="Number of locks held by readers and writers",
+)
+
+wt_cache_ratio = mongodb_timeseries(
+    "WiredTiger Engine Cache hit/miss ratio",
+    [
+        Target(
+            expr=f'(\n   rate(mongodb_ss_wt_cache_pages_requested_from_the_cache{{rs_state="{RS_STATE_PRIMARY}",rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}[$__rate_interval]) - \n   rate(mongodb_ss_wt_cache_pages_read_into_cache{{rs_state="{RS_STATE_PRIMARY}",rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}[$__rate_interval])) / \n   rate(mongodb_ss_wt_cache_pages_requested_from_the_cache{{rs_state="{RS_STATE_PRIMARY}",rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}[$__rate_interval])',
```

Comment:
Should be simplified (and wrapped), something like:
```suggestion
            expr=f'1 -  rate(mongodb_ss_wt_cache_pages_read_into_cache{{rs_state="{RS_STATE_PRIMARY}",rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}[$__rate_interval])
                        /
                        rate(mongodb_ss_wt_cache_pages_requested_from_the_cache{{rs_state="{RS_STATE_PRIMARY}",rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}[$__rate_interval])',
```